server/server.go

```diff
@@ -221,6 +252,30 @@ func Main(keyP crypki.KeyIDProcessor) {
 
 }
 
+type hcHandler struct {
+	hcService *healthcheck.HealthCheckService
+}
+
+func (h *hcHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	if (r.URL.Path != "/ruok" && r.URL.Path != "status") || r.Method != "GET" {
```

Comment:
Not sure if there's a difference from slash...
```suggestion
	if (r.URL.Path != "/ruok" && r.URL.Path != "/status") || r.Method != "GET" {
```